test/uploadDockerImages.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+image=${1}
+sshhost=${2}
+sshkey=${3}
+
+#docker save docker.io/centos:centos7 | bzip2 | ssh -i /home/jchaloup/.ssh/libra.pem ec2-user@ec2-34-200-213-218.compute-1.amazonaws.com 'bunzip2 > /tmp/sideimage && sudo docker image import /tmp/sideimage docker.io/centos:centos7'
+docker save ${image} | bzip2 | ssh -i ${sshkey} ec2-user@${sshhost} "bunzip2 > /tmp/tempimage.bz2 && sudo docker load -i /tmp/tempimage.bz2"
```

Comment:
Why do we need this file?

Reply:
Testing purposes so we can upload docker images to newly provisioned master instances when testing changes in the aws actuator code.

Reply:
But isn't it also implemented in go code?

Reply:
It is though when you run the e2e locally over a cluster deployed in aws you need to upload the locally build aws actuator image to the master machine so you can run the e2e.

Reply:
Seems legit. Can we have that in some readme file? :smile: